photutils/background/core.py

```diff
@@ -0,0 +1,675 @@
+        ``sigma_lower`` and ``sigma_upper``, if input. Defaults to 3.
+    sigma_lower : float or `None`, optional
+        The number of standard deviations to use as the lower bound for
+        the clipping limit. If `None` then the value of ``sigma`` is
+        used. Defaults to `None`.
+    sigma_upper : float or `None`, optional
+        The number of standard deviations to use as the upper bound for
+        the clipping limit. If `None` then the value of ``sigma`` is
+        used. Defaults to `None`.
+    iters : int or `None`, optional
+        The number of iterations to perform sigma clipping, or `None` to
+        clip until convergence is achieved (i.e., continue until the
+        last iteration clips nothing). Defaults to 5.
+    """
+
+    def __call__(self, data):
+        return self.calc_background_rms(data)
+
+    @abc.abstractmethod
+    def calc_background_rms(self, data):
+        """
+        Calculate the background rms value.
+
+        Parameters
+        ----------
+        data : array_like or `~numpy.ma.MaskedArray`
+            The array for which to calculate the background rms value.
+
+        Returns
+        -------
+        result : float
+            The calculated background rms value.
+        """
+
+
+class MeanBackground(BackgroundBase):
+    """
+    Class to calculate the background in an array as the (sigma-clipped)
+    mean.
+
+    Parameters
+    ----------
+    sigma : float, optional
+        The number of standard deviations to use for both the lower and
+        upper clipping limit. These limits are overridden by
+        ``sigma_lower`` and ``sigma_upper``, if input. Defaults to 3.
+    sigma_lower : float or `None`, optional
+        The number of standard deviations to use as the lower bound for
+        the clipping limit. If `None` then the value of ``sigma`` is
+        used. Defaults to `None`.
+    sigma_upper : float or `None`, optional
+        The number of standard deviations to use as the upper bound for
+        the clipping limit. If `None` then the value of ``sigma`` is
+        used. Defaults to `None`.
+    iters : int or `None`, optional
+        The number of iterations to perform sigma clipping, or `None` to
+        clip until convergence is achieved (i.e., continue until the
+        last iteration clips nothing). Defaults to 5.
+
+    Examples
+    --------
+    >>> from photutils import MeanBackground
+    >>> data = np.arange(100)
+    >>> bkg = MeanBackground(sigma=3.)
+
+    The background value can be calculated by using the
+    ``.calc_background()`` method, e.g.:
+
+    >>> bkg_value = bkg.calc_background(data)
+    >>> print(bkg_value)    # doctest: +FLOAT_CMP
+    49.5
+
+    Alternatively, the background value can be calculated by calling the
+    class instance as a function, e.g.:
+
+    >>> bkg_value = bkg(data)
+    >>> print(bkg_value)    # doctest: +FLOAT_CMP
+    49.5
+    """
+
+    def __init__(self, **kwargs):
```

Comment:
This isn't really necessary, right?  That is, if you don't override the `__init__` at all it will just do this?